index.html

```diff
@@ -68,7 +69,204 @@ <h2> I’m a software developer</h2>
+          <h2>Multi-Post Stories</h2>
+          <ul class="portfolio-ul">
+            <li class="portfolio-li">Ruby on rails</li>
+            <li class="portfolio-li">css</li>
+            <li class="portfolio-li">Javascript</li>
+            <li class="portfolio-li">HTML</li>
+          </ul>
+          <button class="portfolio-btn" type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="portfolio-item">
+        <img class="portfolio-img" src="images/Laptop.jpg" alt>
+        <div class="portfolio-details">
+          <h2>Multi-Post Stories</h2>
+          <ul class="portfolio-ul">
+            <li class="portfolio-li">Ruby on rails</li>
+            <li class="portfolio-li">css</li>
+            <li class="portfolio-li">Javascript</li>
+            <li class="portfolio-li">HTML</li>
+          </ul>
+          <button class="portfolio-btn" type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="about-me">
+        <img class="about-me-img" src="./images/illustration-2-about-me-mobile.svg" alt="" />
+        <h3>About Me</h3>
+        <p class="description">I can help you build a product , feature or website Look through some of my work
+          and experience! If you like
+          what you
+          see and have a project you need coded,
+          don’t hestiate to contact me.</p>
+        <p class="resume-btn">Get my Resume</p>
+      </div>
+    </section>
+    <section class="portfolio-lang">
+      <div class="portfolio-globe">
+        <div class="globe-img"><img class="globe-image" src="images/Icon-globe.svg" alt="globe-img"></div>
+        <h3>Languages</h3>
+        <ul class="globe-ul">
+          <li class="globe-li">Javascript</li>
+          <li class="globe-li">Ruby</li>
+          <li class="globe-li">Html</li>
+          <li class="globe-li">css</li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="portfolio-lang">
+      <div class="portfolio-globe">
+        <div class="globe-img"><img class="globe-image" src="images/Icon-frameworks.svg" alt="globe-img"></div>
+        <h3>Frameworks</h3>
+        <ul class="globe-ul">
+          <li class="globe-li">Bootstrap</li>
+          <li class="globe-li">Ruby on rails</li>
+          <li class="globe-li">RSpec</li>
+          <li class="globe-li">Capybara</li>
+          <li class="globe-li">Selenium</li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="portfolio-lang">
+      <div class="portfolio-globe">
+        <div class="globe-img"><img class="globe-image" src="images/Icon-skill.svg" alt=""></div>
+        <h3>Skills</h3>
+        <ul class="globe-ul">
+          <li class="globe-li">Codekit</li>
+          <li class="globe-li">Github</li>
+          <li class="globe-li">Codepen</li>
+          <li class="globe-li">Gitlab</li>
+          <li class="globe-li">Terminal</li>
+        </ul>
+      </div>
+    </section>
```

Comment:
- [ ] This is great to add a `<section>` tag for your `About myself` section, but it seems that you have created separates sections for each technology category, We should only have one main container (`<section>` tag) for the `About myself` and this is the container that should wrap all the technology categories instead of having separate section tags 👍 
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` as well